ext/build.sh

```diff
@@ -44,3 +44,12 @@ cmake .
 make
 make install
 popd
+
+# GPS Parser
+pushd /tmp
+git clone https://github.com/ckgt/NemaTode.git
```

Comment:
@rob-baily in the new updated build process should we be cloning an entire git repo or just pull a tar and build that?

Reply:
Odd that I did not notice this before.  This is not ideal but it does not seem like this repo has any releases.  We could try to make sure we get a current commit based on hash that we know is good.

Reply:
Thats what I found as well.  I think this is fine as is for now.

Reply:
Sounds good.